tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/StaticMethods.java

```diff
@@ -649,4 +649,13 @@ public static BDecimal defaultDecimalArgs(BDecimal a, BDecimal b) {
         BDecimal c = a.add(b);
         return c;
     }
+
+    public static Object defaultDecimalArgs2(String s, BDecimal d) {
```

Comment:
Shall we not name functions as 1 and 2? We can say the previous function as `defaultDecimalArgsAddition` and the second one as `defaultDecimalArgsMultiplication`? I think it is best to give some meaning to the name